appium-demos/pom.xml

```diff
@@ -51,6 +51,24 @@
       <version>5.5.0-M1</version>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>org.testng</groupId>
+          <artifactId>testng</artifactId>
+          <version>6.10</version>
+          <scope>test</scope>
+      </dependency>
+    <dependency>
+      <groupId>junit</groupId>
```

Comment:
Please use JUnit5. We're doing a workshop so we should use the latest version of every framework.